### DISCUSSION TITLE
Is there a way to stream AppleTV movies via Sunshine?

### DISCUSSION THREAD

**Discussion by RacOne** [LizardByte/community#590] [Sunshine]
### Select Topic Area

Question

### Body

Hi everyone,
I am using Sunshine and Moonlight to stream my desktop, and I have noticed that AppleTV movies appear as a black screen during playback. I suspect it has to do with Apple's copy protection, but is there a configuration in Sunshine that can handle this? Or do you have any other advice? I just want to watch the movies on my Android device via local network.

**Comment by ReenigneArcher** [comment]
DRM content is not usually able to be captured.

[a comment by an automated account is not shown]
